LAMPAPI/Login.php

```diff
@@ -12,23 +12,26 @@
 use Contactical\ErrorHandler;
 use Contactical\Error;
 
+// Removed so body is accepted
 // Ensure it's a GET Request
-ensureGET();
+// ensureGET();

```

Comment:
```suggestion
ensurePUT();
```

This will just provide an error if we try to use a GET method again.